types/yog-ral/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-object-literal-type-assertion": false,
+        "only-arrow-functions": false,
+        "no-unnecessary-generics": false

```

Comment:
Please remove these. In general it's good practice to not have these, and in this case these aren't for the definition file itself.

Reply:
ok， i'll remove these